xmtp_api_grpc/src/lib.rs

```diff
@@ -41,7 +38,15 @@ pub async fn create_tls_channel(address: String, limit: u64) -> Result<Channel,
         // Purpose: Specifies how long the client will wait for a response to a keep-alive ping before considering the connection dead.
         // Functionality: If a ping response is not received within this duration, the connection is presumed to be lost and is closed.
         // Impact: This setting is crucial for quickly detecting unresponsive connections and freeing up resources associated with them. It ensures that the client has up-to-date information on the status of connections and can react accordingly.
-        .keep_alive_timeout(Duration::from_secs(25))
+        .keep_alive_timeout(Duration::from_secs(10))
+        .http2_keep_alive_interval(Duration::from_secs(15))
```

Comment:
I can't entirely remember why we removed this. But I believe it was a Converse bug that Rich and I debugged years ago. Here is some of the context https://github.com/xmtp/libxmtp/pull/552/files.

Reply:
Maybe this will fix node and RN will be okay as well... It does make me nervous this close to launch.

Reply:
Okay looking through the tests... That was a V2 issue. Lets bring it back.